R/utils-time.R

```diff
@@ -145,3 +145,20 @@ boundDate <- function(date, dateRange, padDays=1){
   return(date)
 }
 
+#' Format Time as a label with the UTC Offset
+#'
+#' @description Formats a time datum as a text
+#' label including the UTC offset at the end.
+#' @param time The time datum to format as a text label
+formatUTCTimeLabel <- function(time){
+  formatted_time <- ""
+  
+  if(!isEmptyOrBlank(time)){
+    tzf <- format(as.POSIXct(time), "%z")
```

Comment:
It shouldn't ever encounter that in this instance but it's possible that if we use that function elsewhere it might so it's probably a good idea to add a test for it, just in case. I'll do that now